### DISCUSSION TITLE
15

### DISCUSSION THREAD

**Discussion by giscus** [car-ffeine/car-ffeine.github.io#12] [Announcements]
# 15

안녕하세요~

https://car-ffeine.github.io/15

**Comment by BackFoxx** [comment]
역시 서버 개발자가 가장 주의해야 할 것은 N+1이군요 👀 짱이에요

**Comment by drunkenhw** [comment]
대용량의 데이터와 jpa는 맞지 않는 것 같기도 하네요..
글 잘 읽었습니다 👍
